exp/agent/executor/agentExecutor.go

```diff
@@ -0,0 +1,11 @@
+package executor
```

Comment:
go file names should be snake_case. include a doc.go in each package.

Reply:
hahahaha I just made the changes